.mocharc.yml

```diff
@@ -14,3 +14,4 @@ require:
   - tsconfig-paths/register
 recursive: true
 exit: true
+timeout: 3000
```

Comment:
Curious why you needed to increase the test timeout. New tests, or just in general some tests were timing out on you?